Deployment fails on the latest release

With the latest release the deployment (`cdk deploy`) fails due to the following error:

```
Bundling asset GenAIChatBotStack/UserInterface/UserInterfaceDeployment/Asset1/Stage...
src/common/api-client/api-client.ts:109:57 - error TS2554: Expected 0 arguments, but got 1.

109       this._userFeedbackClient = new UserFeedbackClient(this._appConfig);
                                                            ~~~~~~~~~~~~~~~


Found 1 error in src/common/api-client/api-client.ts:109

Error: Command failed: npm --silent --prefix "/Users/angmas/Projects/aws-genai-llm-chatbot/lib/user-interface/react-app" run build
    at checkExecSyncError (node:child_process:885:11)
    at execSync (node:child_process:957:15)
    at Object.tryBundle (/Users/angmas/Projects/aws-genai-llm-chatbot/lib/user-interface/index.ts:184:23)
    at AssetStaging.bundle (/Users/angmas/Projects/aws-genai-llm-chatbot/node_modules/aws-cdk-lib/core/lib/asset-staging.js:2:33)
    at AssetStaging.stageByBundling (/Users/angmas/Projects/aws-genai-llm-chatbot/node_modules/aws-cdk-lib/core/lib/asset-staging.js:1:5297)
    at stageThisAsset (/Users/angmas/Projects/aws-genai-llm-chatbot/node_modules/aws-cdk-lib/core/lib/asset-staging.js:1:2728)
    at Cache.obtain (/Users/angmas/Projects/aws-genai-llm-chatbot/node_modules/aws-cdk-lib/core/lib/private/cache.js:1:242)
    at new AssetStaging (/Users/angmas/Projects/aws-genai-llm-chatbot/node_modules/aws-cdk-lib/core/lib/asset-staging.js:1:3125)
    at new Asset (/Users/angmas/Projects/aws-genai-llm-chatbot/node_modules/aws-cdk-lib/aws-s3-assets/lib/asset.js:1:1080) {
  status: 2,
  signal: null,
  output: [ null, null, null ],
  pid: 42683,
  stdout: null,
  stderr: null
}
```

--- a/lib/user-interface/react-app/src/common/api-client/api-client.ts
+++ b/lib/user-interface/react-app/src/common/api-client/api-client.ts
@@ -105,8 +105,8 @@ export class ApiClient {
   }
 
   public get userFeedback() {
-    if(!this._userFeedbackClient) {
-      this._userFeedbackClient = new UserFeedbackClient(this._appConfig);
+    if (!this._userFeedbackClient) {
+      this._userFeedbackClient = new UserFeedbackClient();
     }
 
     return this._userFeedbackClient;

